rskj-core/src/test/java/co/rsk/peg/federation/FederationChangeIT.java

```diff
@@ -85,11 +94,16 @@ void whenAllActivationsArePresentAndFederationChanges_shouldSuccesfullyChangeFed
         assertPendingFederationIsBuiltAsExpected(pendingFederation);
 
         voteToCommitPendingFederation();
-        var newFederation = federationSupport.getActiveFederation();
+        var newFederationOpt = federationSupport.getProposedFederation();
+        assertTrue(newFederationOpt.isPresent());
+        var newFederation = newFederationOpt.get();
         assertEquals(expectedFederation, newFederation);
         
         // Since Lovell is activated we will commit the proposed federation
-        commitProposedFederation();
+        createSvpFundTx();
```

Comment:
```suggestion
        assertSvpFundTxIsCreatedOnNextCallToUpdateCollections();
```

something like that? wdyt?

Reply:
It is not really an assertion. I think we can leave as is

Reply:
but you are asserting at the end of the method 🤔

Reply:
i see you changed it. Looks v good now :))